yaqd_wright/ndinterp.toml

```diff
@@ -1,6 +1,6 @@
 protocol = "ndinterp"
 doc="yaq daemon representing an NDimensional interpolated offset"
-traits=["has-limits", "is-homeable", "is-daemon"]
+traits=["has-dependents", "has-limits", "is-homeable", "is-daemon"]
```

Comment:
`has-position` to be inserted

Reply:
no change needed

Reply:
has-limits automatically implies has-position, so it is not necessary, but could be more explicit